linalg/hypre.cpp

```diff
@@ -347,6 +698,8 @@ HypreParMatrix::HypreParMatrix(MPI_Comm comm, HYPRE_BigInt glob_size,
 
    /* Make sure that the first entry in each row is the diagonal one. */
    hypre_CSRMatrixReorder(hypre_ParCSRMatrixDiag(A));
+
+   // FIXME:
```

Comment:
Should this be addressed?

Do we support BIGINT + GPU?

Reply:
This, and other places like it, will probably cause issues with bigint enabled.

On the other hand, mixedint may be okay.